datarepo-clienttests/src/main/java/runner/TestRunner.java

```diff
@@ -21,6 +22,9 @@
 import utils.FileUtils;
 import utils.KubernetesClientUtils;
 
+@SuppressFBWarnings(
+    value = "RV_RETURN_VALUE_IGNORED",
+    justification = "We do not use the future of the disrupt script")
```

Comment:
Is there another method of the thread pool that you can call that doesn't expect a result? I thought I remembered there being different variants depending on whether you expected a result or not and wanted to wait for it to finish or not (submit/execute/invoke or something like that).

Reply:
Oh right, yes - I can use execute instead of submit. Thanks!